docs/Passcode/Passcode-template.markdown

```diff
@@ -0,0 +1,131 @@
+# 
+<sub>These materials are for informational purposes only and do not constitute legal advice. You should contact an attorney to obtain advice with respect to the development of a research app and any applicable laws.</sub>
+
+# Passcode
+ResearchKit apps often hold sensitive user data. In order to protect that data, your app can use Apple's Passcode technology from within ResearchKit. With Passcode, your users can provide a four or six digit user-selected code for access or use their fingerprint via Touch ID on devices that provide that feature.
+
+There are several operations that you should know about in order to support Passcode in your app:
+
+ * Creation. This is accomplished using a step where the user enters their desired passcode twice—once to establish it and a second time to confirm it.
+ * Access. After the passcode is created, the app can insist on the four digit passcode or Touch ID.
+ * Editing. The user may want to change the passcode by editing it.
+
+ The passcode is stored in the user's keychain on the device.
+
+##Creating a Passcode
+The creation of a passcode starts with the creation of an `ORKPasscodeStep` object. The `ORKPasscodeStep` class inherits from the `ORKStep` class and provides step functionality that is common in ResearchKit.
+
+The following code demonstrates how to create a passcode step using a four digit passcode:
+
+First, create the step object:
+
+	ORKPasscodeStep *passcodeStep = [[ORKPasscodeStep alloc] init];
+
+Next, set the passcode type. Here, a four digit passcode is used:
+
+	passcodeStep.passcodeType = ORKPasscodeType4Digit;
+
+For additional security, consider using a six digit passcode:
+
+	passcodeStep.passcodeType = ORKPasscodeType6Digit
+
+After the step is created, create an `ORKOrderedTask` object. Pass a string identifier as the first parameter, and the newly created passcode step (along with any additional steps for your app) into an array as the second parameter:
+
+	ORKOrderedTask *task = [[ORKOrderedTask alloc] initWithIdentifier:@"passcodeTask" steps:@[passcodeStep]];
+        
+Pass the newly created task to an `ORKTaskViewController` object and set the view controller's delegate:
+        
+	ORKTaskViewController *taskViewController = [[ORKTaskViewController alloc] initWithTask:task  taskRunUUID:nil];
+	taskViewController.delegate = self
+
+Finally, present the task view controller:
+        
+	[self presentViewController:taskViewController animated:TRUE completion:nil];
+
+The user is instructed to type in a passcode twice—once for the initial passcode, and again to confirm it. Here is the initial view:
+
+<center>
+<img src="passcode.png" style="width: 375px;border: solid black 1px;" \>
```

Comment:
Same issue as with the *Account* screenshots, probably a good idea to recapture the screenshots without the simulator window title bar.